### DISCUSSION TITLE
Can't I change a texture width?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7662] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/128052576-6e09fa31-ff1f-4eba-9ff6-b53fb503b57f.png)

I see this warning in TS.

**Comment by ivanpopelyshev** [comment]
You cant. I suggest to clone() it and change the `tex.frame`  , it should hold link to same rect as `tex.orig` , and width/height is actually taking sizes from it

**Reply by ivanpopelyshev** [reply]
I assume its to make auto-size for new sprites, right? 

Btw, here's width getter: https://github.com/pixijs/pixijs/blob/55da4f04b3d5dba4a0f507550d25cb6335d975fa/packages/core/src/textures/Texture.ts#L667-L670
